src/Diabetes.Core/Models/Food.cs

```diff
@@ -7,12 +7,12 @@
 
 namespace Diabetes.Core.Models
 {
-    
+
     public class Food
     {
         private string _name;
         private string _foodname;
-        private string _foodquantity;
+        private MeasuredAmount _foodquantity;
```

Comment:
There's really no need to call this "foodquantity" since we're in the Food class. Just call it "quantity".